six/modules/c++/six.sicd/source/Utilities.cpp

```diff
@@ -509,6 +510,64 @@ Vector3 Utilities::getGroundPlaneNormal(const ComplexData& data)
     return groundPlaneNormal.unit();
 }
 
+Poly1D Utilities::nPoly(PolyXYZ poly, size_t idx)
+{
+    if (idx >= 3)
+    {
+        throw except::Exception(Ctxt("Idx should be < 3"));
+    }
+
+    std::vector<double> coefs;
+    for (size_t ii = 0; ii < poly.size(); ++ii)
+    {
+        coefs.push_back(poly[ii][idx]);
+    }
+    Poly1D ret(coefs);
+    return ret;
+}
+
+Vector3 Utilities::wgs84Norm(const Vector3& point)
+{
+    scene::WGS84EllipsoidModel model;
```

Comment:
It seems like this should be added as a new method of `EllipsoidModel` instead of being in here... any reason not to?